pkg/service/utils.go

```diff
@@ -33,3 +35,18 @@ func IsInternalRequest(ctx context.Context) bool {
 	b, ok := v.(bool)
 	return ok && b
 }
+
+// getLastLoginTime return latest authenticated time from sessions.
+func getLastLoginTime(sessions []models.KratosSessions) time.Time {
+	var auths []int64
+	for _, s := range sessions {
+		auths = append(auths, s.AuthenticatedAt.UnixMilli())
+	}
+	latest := auths[0]
+	for _, auth := range auths {
```

Comment:
why can't we just do a select max of ( utilize postgres functions ) to achieve this. It would have a better performance as we can avoid couple of iterations and conversions as well.

Reply:
Fixed.